bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Scrollable.java

```diff
@@ -374,7 +374,12 @@ LRESULT WM_VSCROLL (long wParam, long lParam) {
 }
 
 LRESULT wmScrollWheel (boolean update, long wParam, long lParam, boolean horzWheel) {
-	LRESULT result = super.WM_MOUSEWHEEL (wParam, lParam);
+	LRESULT result;
+	if (horzWheel)	{
+		result = super.WM_MOUSEHWHEEL(wParam, lParam);
+	} else {
+		result = super.WM_MOUSEWHEEL (wParam, lParam);
+	}
```

Comment:
@deepika-u 
Logic looks good. One possible improvement I can think of:
LRESULT result = horzWheel ? super.WM_MOUSEHWHEEL(wParam, lParam) : super.WM_MOUSEWHEEL(wParam, lParam);

Reply:
Yes, thanks for the review.
I have updated the code as per your suggestion.